cmake/Dependencies.cmake

```diff
@@ -15,14 +15,14 @@
 ##################################################################
 # CUDA Toolkit libraries (including NVJPEG)
 ##################################################################
-# Note: CUDA 8 support is unofficial.  CUDA 9 is officially supported
-find_package(CUDA 8.0 REQUIRED)
-
-include_directories(${CUDA_INCLUDE_DIRS})
-list(APPEND DALI_LIBS ${CUDA_LIBRARIES})
 
```

Comment:
Remove from the docs `CUDA 8.0 compatibility is provided unofficially.` as well.

Reply:
Done.